Lib/test/test_complex.py

```diff
@@ -99,20 +99,48 @@ def test_truediv(self):
             self.check_div(complex(random(), random()),
                            complex(random(), random()))
 
-        self.assertRaises(ZeroDivisionError, complex.__truediv__, 1+1j, 0+0j)
-        self.assertRaises(OverflowError, pow, 1e200+1j, 1e200+1j)
```

Comment:
Is this check done in another test?

Reply:
Actually, no. I removed this test because it does not have relation to division (and the above test was duplicated).

Moved it into `test_pow`.